Documentation/Migration.md

```diff
@@ -0,0 +1,11 @@
+# Migrating Between Versions
+
+## Migrating from Aardvark 3.x / CoreAardvark 2.x
+
+The biggest change in the Aardvark 4.0 family is the reorganization of files between frameworks. This release introduced two new frameworks: AardvarkMailUI and AardvarkLoggingUI. Both of these frameworks were released with a version number of 1.0. You will likely need to update your dependencies to include one or more of these new frameworks, depending which components of Aardvark you were previously using.
```

Comment:
If there is documentation for these two new modules, can we link to them here? This can be done in a future PR if we don't have that documentation just yet.

Reply:
It might be good to explain the thinking that led to this functionality being moved out as well. A quick sentence explaining that we wanted `Aardvark` to focus on <insert thing here> would help someone rebuild their mental model of our different pods and how they interop

Reply:
> documentation for these two new modules

There's a description of what each module does in the README. Were you thinking something more specific than that?

> It might be good to explain the thinking that led to this functionality being moved out as well.

Yeah I can definitely add that here 👍

Reply:
Addressed in 6cd5a29.